tools/ImportTool.Shared/Spliter.cs

```diff
@@ -0,0 +1,139 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Extensions.Logging;
+
+namespace ImportTool.Shared
+{
+    public static class Spliter
```

Comment:
We can use XXXXCommand pattern for this.

Reply:
Fixed